Dockerfile

```diff
@@ -19,7 +19,7 @@ ENV TZ=America/New_York
 
 RUN sudo apt-get update && sudo apt-get -y upgrade
 
-RUN sudo apt-get install busybox-static fakeroot git tmux dmsetup kpartx netcat-openbsd nmap python-psycopg2 python3-psycopg2 libmagic1 python-lzma python-lzo liblzo2-dev python-six snmp uml-utilities util-linux vlan git unzip curl wget nano postgresql-client socat -y
+RUN sudo apt-get install busybox-static fakeroot git tmux dmsetup kpartx netcat-openbsd nmap python-psycopg2 python3-psycopg2 libmagic1 python-lzma python-lzo liblzo2-dev python-six snmp uml-utilities util-linux vlan git unzip curl wget nano postgresql-client socat nginx -y
```

Comment:
Thanks for the PR.  
Why are you deploying a webserver though?  

You can use local ssh tunnels or any port redirector to allow for access to web-interfaces of the emulated devices.

Reply:
in my opinion that would be additional effort for anyone to setup the web-interface, in this implementation the user can just run docker-compose up -d and the web interface would be live on the go....